modules/auxiliary/dos/http/marked_redos.rb

```diff
@@ -0,0 +1,118 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Dos
+
+  def initialize
+    super(
+      'Name'        => 'marked npm module "heading" ReDoS',
+      'Description' => %q{
+        This module exploits a Regular Expression Denial of Service vulnerability
+        in the npm module "marked". The vulnerable portion of code that this module
+        targets is in the "heading" regular expression. Web applications that use
+        "marked" for generating html from markdown are vulnerable. Versions up to
+        0.4.0 are vulnerable.
+      },
+      'References'  =>
+        [
+          ['URL', 'https://blog.sonatype.com/cve-2017-17461-vulnerable-or-not'],
```

Comment:
`['URL', 'https://github.com/storybooks/marksy/issues/52'],`

Is there a CVE for this issue? CVE-2017-17461 or CVE-2017-17461 ? It looks like the former was rejected.

Reply:
The CVE was rejected by the project at the request of the original reporters; we later found that the originally reported payload would not exploit the issue, but that a different payload we developed internally could.  We worked with the project to get it fixed but never received a response on re-opening the CVE for this item, so we identified it internally as distinct.  Given that novel of an explanation, would you still like for me to add CVE-2017-17461 to the module as a reference?